runtime-complexity/memoization.md

```diff
@@ -1,212 +1,120 @@
 
 ![Fibonacci Number Image](/curriculum/runtime-complexity/34_21-FibonacciBlocks.png)
 
 Pause on slide 9 and write the code to implement the Fibonacci Sequence with a while loop.
 
-  
-
 View the solution on slide 10 and compare to your own code.
 
-  
-
-Next try to implement the Fibonacci Sequence using recursion. Really do this before looking at the solution on slide 12!
-
-  
+Next, try to implement the Fibonacci Sequence using recursion. Really do this before looking at the solution on slide 12!
 
 Notice the difference in Time Complexity between using recursion and a while loop.
 
-  
-
-Finally look at the example code of the Fibonacci Sequence using memoization on slide 13 and again notice the Time Complexity and Space Complexity.
-
-  
+Finally, look at the example code of the Fibonacci Sequence using memoization on slide 13 and again notice the Time Complexity and Space Complexity.
 
 Take a look at the comparison chart showing runtime complexity and space complexity for different sample size. what is your biggest take away from slides 17 and 18?
 
-  
-
-Use the resources on the next few slides to solidify your understanding. Go back and review any of the vocabulary thta you were unsure about at first after you watch the video and go to the links provided.
-
-  
+Use the resources on the next few slides to solidify your understanding. Go back and review any of the vocabularies that you were unsure about at first after you watch the video and go to the links provided.
 
 ### Common Mistakes / Misconceptions
-
-  
-
 List things that apprentices might not realize, might assume at first, or should avoid.
-
-  
-
 - Remember: It’s best to implement memoization on functions that are pure and have repetitive calculations.
 
-  
-  
-
 ### Guided Practice
-
-  
-
 Work step by step to change a recursion of Fibonacci Sequence to a memoization version:
 
-  
-
 ### Recursion
-
 ```
-
 function fibonacci(num) {
 
 if (num <= 1) return 1;
 
     return fibonacci(num - 1) + fibonacci(num - 2);
 
 }
-
 ```
 
 ### Memoization
-1. How do we need to change the function definition.
-     
-     Change this line to include 'memo' as an argument in the function definition:
-         ```function fibonacci(num)```
-
+1. How do we need to change the function definition?
+    Change this line to include 'memo' as an argument in the function definition:
+     ```function fibonacci(num)```
     What would it look like?
+
     Ans:
-```  function fibonacci(num, memo)```
+    ```function fibonacci(num, memo)```
 
 
  2. How do we define 'memo' in the function which was not defined when we used recursion?
-
     What would you add to the function?
 
     Ans: create a hash table to store the data to be used again when called upon.
```

Comment:
create -> Create

Reply:
Not his job for #570, but thanks for the vigilance Kundan 😊